pkg/system/devices_linux.go

```diff
@@ -0,0 +1,87 @@
+package system
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/opencontainers/runc/libcontainer/configs"
```

Comment:
Just looking at the imports, it seems like something that would belong in `runc/libcontainer`.